kobuki_node/package.xml

```diff
@@ -13,71 +13,62 @@
   <url type="repository">https://github.com/kobuki-base/kobuki_ros</url>
   <url type="website">http://ros.org/wiki/kobuki_node</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <!-- ROS -->
-  <build_depend>roscpp</build_depend>
+  <build_depend>angles</build_depend>
```

Comment:
Could probably turn most of these into the `<depend>` variety. I suspect this was written before that tag became available.

Reply:
I did that for most of the dependencies in 9605787.